### PR TITLE
Script for converting tickets from json to csv

### PR DESCRIPTION
**Pull request by willfenton**
- Created `ticket_to_csv.py`, which converts tickets in JSON format to CSV
- Example usage: `python ticket_to_csv.py --json_file=json/tickets.json --csv_file=csv/output.csv`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -126,4 +128,4 @@ venv.bak/
 dmypy.json
 
 # Pyre type checker
-.pyre/
```
Comment (mlopatka) [inline .gitignore]: Add new line at EOF

File: ticket_to_csv.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import json
+import csv
+import argparse
+import pathlib
+
+
+# text preprocessing pipeline
```
Comment (mlopatka) [inline ticket_to_csv.py]: Please add function docstring.

File: ticket_to_csv.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+def preprocess_text(text):
+    text = text.replace(",", "")  # remove commas
+    text = text.replace('"', "")  # remove double quotes
```
Comment (mlopatka) [inline ticket_to_csv.py]: Please include removal of single quotes and handle similarly as double quotes.

File: ticket_to_csv.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    return text
+
+# write a ticket as a row in the CSV
+def write_ticket(csv_writer, ticket, taggers):
```
Comment (mlopatka) [inline ticket_to_csv.py]: Please add a docstring.

File: ticket_to_csv.py [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+    # CSV writer takes a list as a row
+    ticket_list = [ticket_id, ticket_title, ticket_content]
+
+    # get tags for each tagger
+    tag_dict = ticket["tags"]
+    ticket_taggers = tag_dict.keys()
+    for tagger in taggers:
+        if tagger in ticket_taggers:
+            tags = "|".join(tag_dict[tagger])   # using | as a delimeter for now
```
Comment (mlopatka) [inline ticket_to_csv.py]: Will this result in a leading pipe character before the first tag in this field?
Please verify correctness against expected behaviour Maybe write a small test case exercising this code with builtin examples of tag sets.

Comment (willfenton) [reply ticket_to_csv.py]: The join function concatenates the strings in a list with the join character, so there won't be leading or trailing delimeter characters

**Pull request merged**